File: Data Structures & Algorithms/top-k-elements-in-list/submission-4.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} k
     * @return {number[]}
     */
    topKFrequent(nums, k) {
        let counter = new Map;
        for(let num of nums) {
            if(counter.has(num)) counter.set(num, counter.get(num) + 1);
            else counter.set(num, 1);
        }

        let result = [];        
        let values = [...counter.entries()];
        values.sort((a, b) => b[1] - a[1]);
        for (let i = 0; i < k; i++) {
            result.push(values[i][0]);
        }

        return result;
    }
}

File: Data Structures & Algorithms/is-anagram/submission-4.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        #return Counter(s) == Counter(t)
        if len(s) != len(t):
            return False

        countLetter = {}

        for letter in s:
            countLetter[letter] = countLetter.get(letter, 0) + 1

        for letter in t:
            if letter not in countLetter or countLetter[letter] <= 0:
                return False

            countLetter[letter] -= 1

        return True
            